test/run-GW-test.sh

```diff
@@ -0,0 +1,274 @@
+#!/bin/sh
+#set -ex
+usage()
+{
+echo "Usage: $cmd [Options] [<database name>] [wizard_id:passwd]
+Hardcoded curl tests on input geneweb <database name>
+and ultimately check that no failures in $GWDLOG
+Need to properly set 'hardcoded vars' in script header.
+Options:
+-c  to test as cgi and not local port.
+-f  file to be sourced in to overwrite hardcoded vars
+-h  To display this help.
+Use Cases:
+$cmd (with no parameters)
+    search $setenv_file in current folder,
+    if found will be sourced in to overwrite hardcoded vars.
+    else will use all default vars.
+$cmd -f <file with my own hardcoded vars>
+    search specified file
+    if found will be sourced in to overwrite hardcoded vars.
+    else will stop on error as file not found.
+$cmd -f <file with my own hardcoded vars> <database name> <wizard_id:passwd>
+    search specified file
+    if found will be sourced in to overwrite hardcoded vars.
+    else will stop on error as file not found.
+    The provided <database name> will overwrite the DBNAME var.
+    The provided wizard_id:passwd will overwrite the PWD var.
+"
+exit 1
+}
+
+setenv_file="./test-gw-vars.txt"
+
+#=== hardcoded vars (start) ===
+# assumes we are running in the repo folder
+# ./test/run-GW-test.sh
+DBNAME='mydbname'   # database name associated to specified hardcoded vars
+PWD=                # default is without wizard_id:passwd
+
+GWD2START=1
+BASES_DIR="$HOME/Genea/GeneWeb-Bases"
+DIST_DIR="./distribution"
+BIN_DIR="$DIST_DIR/gw"
+GWDLOG=./distribution/gw/gwd.log
+GWCGI=gwd.cgi # the cgi script name that call gwd with cgi parameter
+GWDLOGCGI=/tmp/gwd.log # associated error log
+CLEANLOG=1
+CRLMAXTIME=5
+FAILING_CONDITIONS='CRITICAL|ERROR|Failed'
+WARNING_CONDITIONS='WARNING'
+GREPOPT='-q'
+
+# this is the data for specific persons
+# for synonym test there should be several occurrences of FN+SN
+WIZ=hg
+FN=henri
+SN=gouraud
+OC=0
+ID=1711 # individual Id, ideally should have multiple events
+FID=597 # family id for this individual, ideally, should have multiple families
+IMG_C="alain.0.de_fouchier.jpg" # une image du carrousel de $ID!
+IMG_C_S="alain.0.boucher.jpg" # une image sauvegardée du carrousel de $ID!
+IMG_SRC="famille-h-gouraud.jpg" # une image dans bases/src/mabase/images
+TXT_SRC="famille-h-gouraud.txt" # une source dans bases/src/mabase
+IMG_IM="henri.0.gouraud.jpg" # un portrait dans bases/images/mabase
+# someone without grand parents
+FN1=paul
+OC1=1
+SN1=cosse
+# someone without parents
+FN2=antoine
+OC2=0
+SN2=cosse
+NOTE="chantal" # one specific note
+#=== hardcoded vars (end)   ===
```

Comment:
? @hgouraud

Reply:
This is indeed specific to my base and is an example of the content of "test-gw-vars.txt" which can be supplied for another base.